src/renderers/webgl-fallback/WebGLBackend.js

```diff
@@ -2126,7 +2128,9 @@ class WebGLBackend extends Backend {
 
 							} else {
 
-								gl.framebufferTexture2D( gl.FRAMEBUFFER, attachment, gl.TEXTURE_2D, textureData.textureGPU, 0 );
+								const mipLevel = this.renderer._activeMipmapLevel;
+
+								gl.framebufferTexture2D( gl.FRAMEBUFFER, attachment, gl.TEXTURE_2D, textureData.textureGPU, mipLevel );
```

Comment:
The changes to `gl.framebufferTexture2D()` and `gl.framebufferTextureLayer()` make sure the framebuffers for each mipmap are correctly coupled to the texture's mip levels.